tests/models/unit_cases/case_1a/src/run_1a.py

```diff
@@ -125,8 +125,9 @@ class QTHProblem(
 
 
 if __name__ == "__main__":
+    # sol = run_optimization_problem(HeatProblemTvar)
     sol = run_optimization_problem(
-        HeatProblemTvar,
+        HeatProblem,
```

Comment:
Should HeatProblem or HeatProblemTvar be used in the optim? Maybe deleted the commented out code if it is not needed

Reply:
Doesn't matter, depends on what you try to run locally.